chromium_src/components/sync/base/model_type_unittest.cc

```diff
@@ -14,7 +14,7 @@ TEST_F(ModelTypeTest, EncryptableUserTypes) {
 
 // This test is supposed to fail when sync types are increased/decreased
 TEST_F(ModelTypeTest, ModelTypeCounts) {
-  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 56);
+  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 57);
```

Comment:
New datatype is `kPowerBookmark`.
https://source.chromium.org/chromium/chromium/src/+/main:components/power_bookmarks/core/proto/power_bookmark_meta.proto;bpv=1;bpt=1

Reply:
https://github.com/brave/brave-browser/issues/27928